File: day_01_-_Hanoi_Tower.py
def hanoi(height, fro='left', to='right', through='middle'):
	if height:
		hanoi(height - 1, fro, through, to )
		print fro, ' => ', to
		hanoi(height -1, through, to, fro)

if __name__ == '__main__':
	hanoi(5)

File: Day_01_-_Hanoi_Tower.c
#include <stdio.h>

void hanoi(int, char *, char *, char *);

int main(int argc, char const *argv[]){
	hanoi( atoi(argv[1]) , "left", "right", "middle" );
	return 0;
}

void hanoi(int height, char * from, char * to, char * through) {
	if(height) {
		hanoi(height - 1, from, through, to);
		printf("%-7s => %s\n", from, to);
		hanoi(height - 1, through, to, from);
	}
}